Sources/FoundationEssentials/FileManager/SearchPaths/FileManager+WindowsSearchPaths.swift

```diff
@@ -60,7 +60,9 @@ func _WindowsSearchPathURL(for directory: FileManager.SearchPathDirectory, in do
         _url(for: FOLDERID_Public)
 
     case (.trashDirectory, .userDomainMask):
-        _url(for: FOLDERID_RecycleBinFolder)
+        // The "Recycle Bin" is a virtual folder and we cannot get a path from
+        // it directly using `SHGetKnownFolderPath`.
+        nil
```

Comment:
Is this a permanent change (i.e. expected behavior) for Windows or something we want to come back and fix later? If it's "permanent" we could just remove this case altogether and let it fallback to the `default`.

Additionally, `FileManagerTests.testSearchPaths` currently asserts that `(.trashDirectory, .userDomainMask)` produces a directory on Windows. Should we update the test expectations for this change? I think to resolve it we'd just need to add a

```swift
#if os(Windows)
let isWindows = true
#else
let isWindows = false
#endif
```

to the test and then change the `(isMacOS && isFramework) || !isDarwin` conditional to `(isMacOS && isFramework) || (!isDarwin && !isWindows)` to ensure it validates that we don't produce a directory on Windows

Reply:
Ok sounds good then leaving the explicit case / `return nil` with the comment makes sense to me (maybe we should add a "TODO:" to the comment just so it's a bit more visible as something we want to come back to?

Reply:
Sure, I'll add a comment.